kvs/src/kvs/utils.cpp

```diff
@@ -30,6 +30,8 @@ void send_gossip(AddressKeysetMap& addr_keyset_map, SocketCache& pushers,
       if (stored_key_map.find(key) == stored_key_map.end()) {
         // we don't have this key stored, so skip
         continue;
+      } else if (is_metadata(key)) {
+        type = LWW;
```

Comment:
Is this check necessary? I think metadata should be already in the `stored_key_map` with `LWW` as the type right?

Reply:
Ah, okay. Good catch.